### PR TITLE
VSTS: Add yml build deffinition

### PR DESCRIPTION
**Pull request by dburner**
- added yml build deffinition for VSTS

### CODE REVIEW

File: .vsts-ci.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+- task: NuGetCommand@2
+   displayName: "Create nuget packages"
+  inputs:
+    command: 'pack'
+    includeReferencedProjects: 'true'
+    includeSymbols: 'true'
+
+- task: PublishBuildArtifacts@1
+  displayName: "Publish symbols"
```
Comment (alexandra-olteanu) [inline .vsts-ci.yml]: Rename this to publish packages. and please update the ArtifactName to something else.. maybe "Packages"..

File: .vsts-ci.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - visualstudio
+    - vstest
+
+variables:
+  Solution: "Community.Activities.sln"
+  BuildConfiguration: "Release"
+  BuildPlatform: "Any CPU"
+
+steps:
+
```
Comment (alexandra-olteanu) [inline .vsts-ci.yml]: Please add a Build Versioning task. You can inspire from the current Appveyor "Init script" that updates the script. and then add " versioningScheme: "byBuildNumber" as input in nuget pack task.

File: .vsts-ci.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- task: VSTest@2
+  displayName: "Run unit tests"
+  inputs:
+    testAssemblyVer2: |
+          **\$(BuildConfiguration)\*test*.dll
+          !**\obj\**
+    platform: '$(BuildPlatform)'
+    configuration: '$(BuildConfiguration)'
+
+- task: PublishSymbols@1
```
[the remark this answers is not in the document]
Comment (dburner) [reply .vsts-ci.yml]: In the releases section the symbols appear also so I think they should remain in the same artifact.

[the remark this answers is not in the document]
Comment (alessandromuresan) [reply .vsts-ci.yml]: @alexandra-olteanu why separate them? We use the same artifact packaging process for all projects. The `Setup` artifact is named after the folder in which all packaging results are dumped. Whether or not all its contents are used in the deployment process is of no concern at build time.

**Pull request merged**